internal/pkg/aws/cloudwatch/cloudwatch.go

```diff
@@ -0,0 +1,140 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package cloudwatch provides a client to make API requests to Amazon CloudWatch Service.
+package cloudwatch
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy/cloudformation/stack"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+)
+
+const (
+	compositeAlarmType = "Composite"
+	metricAlarmType    = "Metric"
+)
+
+type cwClient interface {
+	DescribeAlarms(input *cloudwatch.DescribeAlarmsInput) (*cloudwatch.DescribeAlarmsOutput, error)
+	ListTagsForResource(input *cloudwatch.ListTagsForResourceInput) (*cloudwatch.ListTagsForResourceOutput, error)
+}
+
+// CloudWatch wraps an Amazon CloudWatch client.
+type CloudWatch struct {
+	client cwClient
+}
+
+// App contains basic info for an application.
+type App struct {
+	AppName     string
+	EnvName     string
+	ProjectName string
+}
+
+// AlarmStatus contains CloudWatch alarm status.
+type AlarmStatus struct {
+	Arn          string
+	Name         string
+	Reason       string
+	Status       string
+	Type         string
+	UpdatedTimes int64
+}
+
+// New returns a CloudWatch struct configured against the input session.
+func New(s *session.Session) *CloudWatch {
+	return &CloudWatch{
+		client: cloudwatch.New(s),
+	}
+}
+
+// GetAlarms returns all CloudWatch alarms associated with the app.
+func (c *CloudWatch) GetAlarms(app App) ([]AlarmStatus, error) {
```

Comment:
The `aws/*` packages shouldn't have any inputs or outputs that are CLI specific. 
What do you think of instead?
```suggestion
// GetAlarmsWithTags returns all the CloudWatch alarms that have the resource tags.
func (c *CloudWatch) GetAlarmsWithTags(tags map[string]string) ([]AlarmStatus, error)
```